src/content/docs/zh-cn/guides/troubleshooting.mdx

```diff
@@ -1,28 +1,18 @@
 ---
-title: 故障排查
+title: 故障排除
 description: 需要帮助吗？卡在什么地方了？我们已经帮你解决了。
+i18nReady: true
 ---
 
 Astro 提供了几个不同的工具来帮助你排除故障和调试你的代码。
 
 ## 常见错误信息
 
-下面是你在终端可能看到的一些常见错误信息，它们可能意味着什么，以及如何处理它们。
-
-### 转化失败，出现 X 错误
-
-这条信息经常出现，因为目前 Astro 要求导入和导出语句必须在 `.astro`文件顶层。
-
-**解决方案**：把你的导入和导出语句写在你的组件脚本的顶部。
-
-**状态**：现有限制，正在进行修复。
-
-**不确定这是否是你的问题？**
-检查一下是否有其他人报告过[这个问题](https://github.com/withastro/astro/issues?q=is%3Aissue+is%3Aopen+Transform+failed+with+*+error)!
+下面是你在终端可能会看到的一些常见错误信息，包括它们可能的含义及解决方案。
 
 ### 不能在模块外使用导入语句
```

Comment:
I think we should keep the original English error message, don't you?
```suggestion
### Cannot use import statement outside a module（不能在模块外使用导入语句）
```

Reply:
Agreed. It'll be easier to debug. I'll fix that.

Reply:
You can use github's commit suggestion feature directly. It works very well.